openshift/topics/get_started.adoc

```diff
@@ -19,7 +20,8 @@ To create a Registry Service Account, navigate to the link:https://access.redhat
 . Finally, click the *_OpenShift Secret_* tab, and perform all steps listed on that page.
 
 See the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] article for more information.
-====
+
+.Procedure
 
 . Ensure that you are logged in as a cluster administrator or a user with project administrator access to the global openshift project:
```

Comment:
Replace "to the global openshift project" with "to the global `openshift` project" (since `openshift` is the reserved name, not just ordinary project. IOW that `openshift` name should be highlighted).